raiden_contracts/tests/fixtures/channel.py

```diff
@@ -732,34 +737,41 @@ def get(
     return get
 
 
-def call_settle(token_network, channel_identifier, A, vals_A, B, vals_B):
-    A_total_transferred = vals_A.transferred + vals_A.locked
-    B_total_transferred = vals_B.transferred + vals_B.locked
+def call_settle(
+        token_network,
+        channel_identifier,
+        A,
+        vals_A: ChannelValues,
+        B,
+        vals_B: ChannelValues,
+):
+    A_total_transferred = vals_A.transferred + vals_A.locked_amounts.locked
+    B_total_transferred = vals_B.transferred + vals_B.locked_amounts.locked
     assert B_total_transferred >= A_total_transferred
 
     if A_total_transferred != B_total_transferred:
         with pytest.raises(TransactionFailed):
             token_network.functions.settleChannel(
-                channel_identifier,
-                B,
-                vals_B.transferred,
-                vals_B.locked,
-                vals_B.locksroot,
-                A,
-                vals_A.transferred,
-                vals_A.locked,
-                vals_A.locksroot,
+                channel_identifier=channel_identifier,
+                participant1=B,
+                participant1_transferred_amount=0,
```

Comment:
why are you not using `transferred` anymore?

Reply:
oops.  That's a bug.